domain-rrd/rrd-api-controller/src/main/java/gov/va/vro/controller/rrd/ClaimMetricsController.java

```diff
@@ -86,7 +86,13 @@ public ResponseEntity<List<ExamOrderInfoResponse>> examOrderSlack(
       Integer page, Integer size, Boolean notOrdered) throws ClaimProcessingException {
     ExamOrderInfoQueryParams params =
         ExamOrderInfoQueryParams.builder().page(page).size(size).notOrdered(notOrdered).build();
-    ExamOrdersInfo examOrdersInfo = claimMetricsService.findExamOrderInfo(params);
+    ExamOrdersInfo examOrdersInfo = claimMetricsService.findExamOrderInfoOlderThan24(params);
+    if (examOrdersInfo.getExamOrderInfoList().size() < 1) {
+      throw new ClaimProcessingException(
```

Comment:
Seems like an odd place to throw a `ClaimProcessingException`. I would probably just return an empty list instead.